ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -0,0 +1,39 @@
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_path: Dockerfile
+  to: openshift-lightspeed-operator
+promotion:
+  to:
+  - name: "4.15"
+    namespace: lightspeed-operator
```

Comment:
Verify if this is correct - light-speed service is using ols as namespace here .

Reply:
I agree, let's publish the artifacts to the same as lightspeed-service repository does.

Reply:
yes, please align this with lightspeed-service.

https://github.com/openshift/release/blob/1a80de2a855bae391be7e49018be32086076b9d8/ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml#L9-L12